src/routes/expense.router.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const express = require('express');
```

Comment:
Consider using `import` instead of `require` for importing modules to follow the ES6 module syntax. Example: import express from 'express';